src/Spryker/Glue/AppKernel/Validator/BodyStructureValidator.php

```diff
@@ -35,38 +36,57 @@ public function validate(GlueRequestTransfer $glueRequestTransfer): GlueRequestV
             ->setIsValid(true)
             ->setStatus(Response::HTTP_OK);
 
-        $content = $this->appKernelToUtilEncodingService->decodeJson((string)$glueRequestTransfer->getContent(), true);
-        $constraintViolationList = $this->validator->validate($content, $this->getConstraintForRequestStructure());
+        try {
+            $content = $this->appKernelToUtilEncodingService->decodeJson((string)$glueRequestTransfer->getContent(), true);
+        } catch (Throwable $throwable) {
+            return $this->getFailedGlueRequestValidationTransfer($glueRequestValidationTransfer);
+        }
 
-        if ($constraintViolationList->count() > 0) {
-            $glueRequestValidationTransfer
-                ->setIsValid(false)
-                ->setStatus(Response::HTTP_UNPROCESSABLE_ENTITY);
+        $constraintViolationList = $this->validator->validate(['content' => $content], $this->getConstraintForRequestStructure());
 
-            $glueRequestValidationTransfer->addError(
-                (new GlueErrorTransfer())
-                    ->setCode((string)Response::HTTP_UNPROCESSABLE_ENTITY)
-                    ->setMessage(AppKernelConfig::RESPONSE_MESSAGE_VALIDATION_FORMAT_ERROR_MESSAGE),
-            );
+        if ($constraintViolationList->count() > 0) {
+            return $this->getFailedGlueRequestValidationTransfer($glueRequestValidationTransfer);
         }
 
         return $glueRequestValidationTransfer;
     }
 
+    protected function getFailedGlueRequestValidationTransfer(
+        GlueRequestValidationTransfer $glueRequestValidationTransfer
+    ): GlueRequestValidationTransfer {
+        $glueRequestValidationTransfer
+            ->setIsValid(false)
+            ->setStatus(Response::HTTP_UNPROCESSABLE_ENTITY);
+
+        $glueRequestValidationTransfer->addError(
+            (new GlueErrorTransfer())
+                ->setCode((string)Response::HTTP_UNPROCESSABLE_ENTITY)
+                ->setMessage(AppKernelConfig::RESPONSE_MESSAGE_VALIDATION_FORMAT_ERROR_MESSAGE),
+        );
+
+        return $glueRequestValidationTransfer;
+    }
+
     protected function getConstraintForRequestStructure(): Collection
     {
         return new Collection([
-            'data' => new Collection([
-                'type' => new EqualTo(AppKernelConfig::REQUEST_DATA_TYPE),
-                'attributes' => new Collection([
-                    'configuration' => [
-                        new Required(),
-                        new NotBlank(),
-                        new Type(['type' => 'string']),
-                        new Json(),
-                    ],
+            'content' => [
```

Comment:
Why is the format changed?

Reply:
Because if the response is not in JSON format or is empty, the previous check would have passed.